secure_message/services/party_service.py

```diff
@@ -10,18 +10,27 @@
 
 
 class PartyService:
+
+    __users_cache = dict()
```

Comment:
What is the reason for making these private? Generally the only use case for dunderscores is the following:

"If your class is intended to be subclassed, and you have attributes that you do not want subclasses to use, consider naming them with double leading underscores and no trailing underscores. This invokes Python's name mangling algorithm, where the name of the class is mangled into the attribute name. This helps avoid attribute name collisions should subclasses inadvertently contain attributes with the same name." - see [pep8](https://www.python.org/dev/peps/pep-0008/#designing-for-inheritance) for more details